lib/commands/new/project-template.js

```diff
@@ -64,6 +66,13 @@ exports.ProjectTemplate = class {
     );
   }
 
+  createServeConfig(){
+    return {
+      localPort:9000,
```

Comment:
can you make sure that the linter is OK with this? I think a space is needed between `localPort:` and `9000`